test/types/record.test.ts

```diff
@@ -49,8 +69,11 @@ const record2ToArray: any[] = [...record2]
 const record1Has: boolean = record1.has(42)
 const record2Has: boolean = record1.has('key')
 
-const record1Get1: any = record1.get(42)
-const record2Get1: any = record2.get('key')
+const record1Get1: any = record1.get('name')
+const record2Get1: any = record2.get('age')
+
+const record1Get2: object = record1.get('name')
+const record2Get2: string[] = record2.get('age')
 
-const record1Get2: object = record1.get(42)
-const record2Get2: string[] = record2.get('key')
```

Comment:
This line should be put be putted back on the tests

Reply:
What should be the value type in this case?

```ts
const value: unknown = record.get('key')
```